tests/nfv/run_integration_tests.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Executes all the integration tests to verify that
+# vsperf can control openvswitch properly
+#
+# Maintainer: Jose Lausuch <jalausuch@suse.com>
+
+use base "consoletest";
+use testapi;
+use strict;
+use utils;
+use lockapi;
+use mmapi;
+
+sub run {
+    my $self        = shift;
+    my $vsperf_conf = "/etc/vsperf_ovs.conf";
+
+    select_console 'root-console';
+    mutex_create('nfv_trafficgen_ready');
+
+    # wait until traffic generator installation finishes
+    wait_for_children;
```

Comment:
Currently you don't have multi-machine tests , and by this single line you will not introduce them. I am suggesting to add everything related to multi-machine tests in single commit. And for now just drop this line

Reply:
The tests are executed in 2 separate machines currently:
https://openqa.suse.de/tests/overview?distri=sle&version=15&build=410.1&groupid=133

Reply:
acknowledged, let's leave it like this